### PR TITLE
Reverts Ghost being votable at any time

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Title, moves ghost down to 40 pop

## Why It's Good For The Game
Every time this story teller is voted pop is slashed by half, peak hours should not be going from 90 connected to 50 connected because 4 people voted ghost, those 4 people can go ERP somewhere else

## Proof Of Testing
it works trust me

## Changelog

:cl:
server: ghost only votable up to 40 pop
/:cl:

### CODE REVIEW

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
the changelog should be server: or admin:
not fix:

File: modular_zubbers/code/modules/storyteller/storytellers/storyteller_tellers.dm [hunk 1]
```diff
@@ -64,4 +64,5 @@
 	name = "The Ghost"
 	desc = "The Ghost is the absence of a Storyteller. It will not spawn a single event of any sort, or run any Antagonists. Best for rounds where the population is so low that not even the Sleeper is low enough."
 	disable_distribution = TRUE
+	population_max = 30
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/storyteller/storytellers/storyteller_tellers.dm:67]:
```suggestion
	population_max = 40
```

**Review by Majkl-J — CHANGES_REQUESTED** [review]
Bump it up to 40

**Review by Majkl-J — APPROVED** [review]

**Review by Majkl-J — APPROVED** [review]

**Comment by NargaInDisguise** [comment]
I was the one that originally suggested moving ghost's pop limit up, so this is more than fine with me, 40 is reasonable

**Pull request merged**